main.js

```diff
@@ -105,7 +104,7 @@ const getAppContainer = (options) => {
 	app.use(vary);
 
 	if (!options.demo) {
-		instrumentListen.addMetrics(healthChecks(app, options, meta));
+		setupHealthEndpoint(app, options, meta);
```

Comment:
**suggestion**: this is now just creating a route, and it's the only thing creating a route that's in another file. i think it would be easier to follow if we inlined it.

Reply:
I agree, was just trying to keep the diff small and readable because a lot is already changing 😁 I'll move it as a separate commit

Reply:
Actually having messed around with this it makes the diff quite messy and I'm not sure if I want my name against it all 😄 is it alright to reconsider the structure separately? There's quite a lot of stuff that could move around